src/containers/explore/EntityDataContainer.js

```diff
@@ -0,0 +1,150 @@
+import { selectOrgDataSet, selectOrganization } from '../../core/redux/selectors';
+import { Routes } from '../../core/router';
+import { clipboardWriteText } from '../../utils';
+import { EntityDataGrid } from './components';
+
+const { getPropertyValue } = DataUtils;
+const { isPending } = ReduxUtils;
+const { isValidUUID } = ValidationUtils;
+
+const FlexContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 30px;
+`;
+
+type Props = {
+  dataSetDataRoute ?:string;
+  dataSetId :UUID;
+  entityKeyId :UUID;
+  isModal ?:boolean;
+  organizationId :UUID;
+  organizationRoute ?:string;
+};
+
+const EntityDataContainer = ({
+  dataSetDataRoute,
+  dataSetId,
+  entityKeyId,
+  isModal = false,
+  organizationId,
+  organizationRoute,
+} :Props) => {
+  const dispatch = useDispatch();
+
+  const linkString = `https://openlattice.com/orgs/#${
+    Routes.ENTITY_DETAILS
+      .replace(Routes.DATA_SET_ID_PARAM, dataSetId)
+      .replace(Routes.ENTITY_KEY_ID_PARAM, entityKeyId)
+      .replace(Routes.ORG_ID_PARAM, organizationId)
+  }`;
+
+  const dataSet :Map<FQN, List> = useSelector(selectOrgDataSet(organizationId, dataSetId));
+  const entityData :?Map = useSelector((s) => s.getIn([EXPLORE, SELECTED_ENTITY_DATA], Map()));
+  const exploreEntityDataRS :?RequestState = useRequestState([EXPLORE, EXPLORE_ENTITY_DATA]);
+  const neighbors :?Map = useSelector((s) => s.getIn([EXPLORE, ENTITY_NEIGHBORS_MAP, entityKeyId], Map()));
+  const organization :?Organization = useSelector(selectOrganization(organizationId));
+
+  const name :string = getPropertyValue(dataSet, [FQNS.OL_DATA_SET_NAME, 0]);
+  const title :string = getPropertyValue(dataSet, [FQNS.OL_TITLE, 0]);
+
+  useEffect(() => {
+    if (isValidUUID(entityKeyId) && isValidUUID(dataSetId)) {
+      dispatch(exploreEntityData({ entityKeyId, entitySetId: dataSetId }));
+      dispatch(exploreEntityNeighbors({ entityKeyId, entitySetId: dataSetId }));
+    }
+  }, [dispatch, entityKeyId, dataSetId]);
+
+  if (isPending(exploreEntityDataRS)) {
+    return (
+      <Spinner size="2x" />
+    );
+  }
+
+  if (entityData && dataSetId && organizationId) {
+    return (
+      <>
+        <AppContentWrapper bgColor="white">
+          {
+            organization && dataSet && !isModal && (
+              <Crumbs>
+                <CrumbLink to={organizationRoute}>{organization.title || 'Organization'}</CrumbLink>
+                <CrumbLink to={dataSetDataRoute}>{title || name}</CrumbLink>
+                <CrumbItem>{entityKeyId}</CrumbItem>
+              </Crumbs>
+            )
+          }
+          <FlexContainer>
+            <Typography gutterBottom variant="h1">{entityKeyId}</Typography>
+            <LinkButton
+                color="default"
+                isDisabled={!linkString.length}
+                onClick={() => clipboardWriteText(linkString)}>
+              Get Link
+            </LinkButton>
+          </FlexContainer>
```

Comment:
The button doesn't look great on smaller screens. 

<img width="947" alt="Screen Shot 2021-06-07 at 7 43 28 PM" src="https://user-images.githubusercontent.com/440299/121114859-2b13bd80-c7c9-11eb-81b7-0ef4e717a6a0.png">

It behaves a little better if you use the existing `SpaceBetweenGrid` component.

```suggestion
          <SpaceBetweenGrid>
            <Typography gutterBottom variant="h1">{entityKeyId}</Typography>
            <LinkButton
                color="default"
                isDisabled={!linkString.length}
                onClick={() => clipboardWriteText(linkString)}>
              Get Link
            </LinkButton>
          </SpaceBetweenGrid>
```